python/dgl/data/citation_graph.py

```diff
@@ -166,160 +464,100 @@ def _sample_mask(idx, l):
     mask[idx] = 1
     return mask
 
-def load_cora():
-    data = CitationGraphDataset('cora')
-    return data
+"""Get CoraGraphDataset
```

Comment:
Function comment should be inside function
def load_cora(...):
    """Get....."""